<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>loggedInFrame</class>
 <widget class="QFrame" name="loggedInFrame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QFrame" name="frame">
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="rightMargin">
         <number>9</number>
        </property>
        <property name="bottomMargin">
         <number>3</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="logOutButton">
            <property name="maximumSize">
             <size>
              <width>40</width>
              <height>30</height>
             </size>
            </property>
            <property name="statusTip">
             <string>Log Out</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>../Img/left_arrow.png</normaloff>../Img/left_arrow.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>24</width>
              <height>24</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="addPasswordButton">
            <property name="maximumSize">
             <size>
              <width>40</width>
              <height>50</height>
             </size>
            </property>
            <property name="statusTip">
             <string>Add Password</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>../Img/plus_sign.png</normaloff>../Img/plus_sign.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>24</width>
              <height>24</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QTableView" name="dataTable">
       <property name="statusTip">
        <string>Passwords</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>logOutButton</tabstop>
  <tabstop>addPasswordButton</tabstop>
  <tabstop>dataTable</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
